renderdoc/driver/vulkan/vk_shaderdebug.cpp

```diff
@@ -4295,6 +4369,30 @@ ShaderDebugTrace *VulkanReplay::DebugPixel(uint32_t eventId, uint32_t x, uint32_
     RDCLOG("useSampleID is %u because of bare capability", useSampleID);
   }
 
+  bool useViewIndex = (view == ~0U) ? false : true;
+  if(useViewIndex)
+  {
+    ResourceId rp = state.GetRenderPass();
+    const VulkanCreationInfo::RenderPass &rpInfo =
+        m_pDriver->GetDebugManager()->GetRenderPassInfo(rp);
```

Comment:
This is not valid if dynamic rendering is in use and there is no renderpass. The viewIndex mask will need to be obtained from the dynamic rendering info.